mods/voidmulti.sc2mod/base.sc2data/gamedata/effectdata.xml

```diff
@@ -724,14 +736,15 @@
         <PeriodicOffsetArray value="0,-11,0"/>
         <PeriodicOffsetArray value="0,-12,0"/>
         <PeriodicEffectArray index="0" value="LurkerMPCU"/>
+        <PeriodicValidator value=""/>
```

Comment:
- Removed CasterIsAttacking validator from Lurker attack's spine splash damage hits
- Added channeled and effectFailure flag to Lurker's attack's persistent effect

Why was this changed? Are there side effects? Does target switching during attacks break something? Do other actions directly after the attack started break something?

Reply:
I've now looked into this. I still have no idea why this was changed, but it has side effects.

- Changing the attack target while the persistent effect is active causes it to end. So, changing the attack target via an order will cancel the current attack!
- When targeting a friendly target, you can now order Stop to cancel the attack mid-travel

I doubt that this is the desired behavior. The changes to this persistent effect need to be reverted.